chsdi/templates/htmlpopup/tiefengeothermie_projekte.mako

```diff
@@ -20,10 +20,27 @@
    <tr><th class="cell-left">${_('ch.swisstopo.geologie-tiefengeothermie_projekte.canton')}</th><td>${c['attributes']['canton'] or '-'}</td></tr>
    <tr><th class="cell-left">${_('ch.swisstopo.geologie-tiefengeothermie_projekte.community')}</th><td>${c['attributes']['community'] or '-'}</td></tr>
    <tr><th class="cell-left">${_('ch.swisstopo.geologie-tiefengeothermie_projekte.depth')}</th><td>${c['attributes']['depth'] or '-'}</td></tr>
+   <tr><th class="cell-left">${_('ch.swisstopo.geologie-tiefengeothermie_projekte.reservoir')}</th><td>${c['attributes']['reservoir'] or '-'}</td></tr>
    <tr><th class="cell-left">${_('ch.swisstopo.geologie-tiefengeothermie_projekte.temp')}</th><td>${c['attributes']['temp'] or '-'}</td></tr>
    <tr><th class="cell-left">${_('ch.swisstopo.geologie-tiefengeothermie_projekte.power')}</th><td>${c['attributes']['power'] or '-'}</td></tr>
    <tr><th class="cell-left">${_('ch.swisstopo.geologie-tiefengeothermie_projekte.produc')}</th><td>${c['attributes']['produc'] or '-'}</td></tr>
-   <tr><th class="cell-left">${_('ch.swisstopo.geologie-tiefengeothermie_projekte.weblink')}</th><td><a href="${c['attributes']['weblink']}" target="_blank">Link</a></td></tr>
-   <tr><th class="cell-left">${_('ch.swisstopo.geologie-tiefengeothermie_projekte.id_project')}</th><td>${c['attributes']['id_project'] or '-'}</td></tr>
+   <tr><th class="cell-left">${_('ch.swisstopo.geologie-tiefengeothermie_projekte.download')}</th>
+   % if c['attributes']['download'] == None or c['attributes']['download'] == "-":
+       <td>-</td>
+   % else:
+       <td><a href="${c['attributes']['download']}" target="_blank">Zip</a></td>
+   % endif
+   </tr>
+   <%
+   weblink = c['attributes']['weblink'].split('; ')
+   %>
+   <tr>
+     <th class="cell-left">${_('ch.swisstopo.geologie-tiefengeothermie_projekte.weblink')}</th>
```

Comment:
Please implement an "or "-"" for those cases where no link is present

Reply:
Done